### PR TITLE
message-system 69: Ask users that installed unintentionally released 2.8.6 FW on TS5 to contact support

### PR DESCRIPTION
**Pull request by matejkriz**
## Description

- Mobile only
- TS5 with 2.8.6 only

Mobile only support English so we don't need translations.

## Screenshot
Screenshot_2024-12-19-11-37-11-038_io trezor suite debug
ts:

### CODE REVIEW

File: suite-common/message-system/config/config.v1.json [hunk 1]
```diff
@@ -1,8 +1,70 @@
...
+                    "tr": "You have downloaded an incomplete firmware version 2.8.6, released unintentionally. Tap here to open our chatbot and create a support ticket for assistance. Remember, never share your wallet backup with anyone under any circumstances.",
+                    "pt": "You have downloaded an incomplete firmware version 2.8.6, released unintentionally. Tap here to open our chatbot and create a support ticket for assistance. Remember, never share your wallet backup with anyone under any circumstances.",
+                    "uk": "You have downloaded an incomplete firmware version 2.8.6, released unintentionally. Tap here to open our chatbot and create a support ticket for assistance. Remember, never share your wallet backup with anyone under any circumstances."
+                },
+                "cta": {
+                    "action": "external-link",
+                    "link": "https://trezor.io/learn/c/trezor-suite-lite#open-chat",
```
Comment (matejkriz) [inline suite-common/message-system/config/config.v1.json]: This link has to be replaced by specific one.

File: suite-common/message-system/config/config.v1.json [hunk 2]
```diff
@@ -1,8 +1,70 @@
...
-    "sequence": 68,
+    "timestamp": "2024-12-19T00:00:00+00:00",
+    "sequence": 69,
     "actions": [
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": "!",
+                        "mobile": "24.12.1",
```
Comment (matejkriz) [inline suite-common/message-system/config/config.v1.json]: We should allow newer version too. And maybe even older? But that sounds like a too much on an edge case.

Comment (matejkriz) [reply suite-common/message-system/config/config.v1.json]: fixed, force pushed

**Review by MiroslavProchazka — APPROVED** [review]

**Review by Lemonexe — APPROVED** [review]
Works as expected locally; on 2.8.6 T3T1 it displays this message, not present on other versions, or other devices:
image

**Comment by STew790** [comment]
**QA OK**
Works on mobile also as expected. Only connected TS5 affected with the correct 2.8.6 FW. Other devices won't trigger this.

image
image

**Pull request merged**